styles.css

```diff
@@ -21,29 +21,53 @@
 }
 
 .nav-menu {
-  align-items: center;
-  font-size: 20px;
-  position: absolute;
-  left: 30px;
-  top: 10px;
-  bottom: 0;
-  width: 375px;
-  height: 40px;
-  overflow: hidden;
-}
-
-.social-nav {
   display: flex;
   flex-direction: row;
+  justify-content: space-between;
   align-items: center;
-  padding: 0;
-  gap: 12px;
-  position: absolute;
-  width: 200px;
-  height: 50px;
-  order: 3;
-  font-size: 20px;
-  margin-top: 10px;
+  padding: 1rem 1.5rem;
+  border-bottom: 1px solid #36b37f;
+}
+
+.welcome {
+  margin-left: 5px;
+}
+
+.menu-item {
+  display: block;
+  position: fixed;
+  top: 5rem;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  left: -100%;
+  border-radius: 10px;
+  transition: 0.3s;
+  box-shadow: 0 10px 27px rgba(0, 0, 0, 0.05);
+}
+
+.menu-item.active {
+  left: 0;
+  color: #3a4a42;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 44px;
+  background: #fff;
+  z-index: 100;
+}
```

Comment:
- [ ]  Great job but I think that If I scroll i will see all the section but depend on the Figma template I should just see the menu without scrolling in the side so kindly fix it to match the Figma.🔥🔥💪🏻💪🏻✅